### ISSUE TITLE
Suppor SQLITE_ENABLE_ICU?

### ISSUE THREAD

**Issue by sergeymild** [clemensg/sqlite3pod#13]
is it possible to add support SQLITE_ENABLE_ICU ?

**Comment by clemensg** [comment]
Yes, you can send a PR or I can look into it when the next release is due.

Not sure if icu libraries are shipped with iOS and if we have to add linker flags as well or if adding the define is enough?

**Comment by clemensg** [comment]
I looked into this some more: Looks like on iOS there is only a reduced version of ICU present, for example the header file `unicode/ucol.h` is missing (not sure how important that one is and if we could add a patch to work around). On macOS there seems to be no ICU shipped at all.
To support this, we'd probably have to link to an externally built version of ICU..

**Comment by clemensg** [comment]
Hi, I added a branch with experimental support. I could only test it on macOS 12.1 so far so it would be great if you could try it out and report back. You can reference the branch like this `pod 'sqlite3', inhibit_warnings: true, subspecs: ['fts5', 'icu'], git: 'https://github.com/clemensg/sqlite3pod.git', branch: 'icu'`
